ClassHierarchyAndCollections/ClassHierarchyAndCollections/School.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ClassHierarchyAndCollections
+{
+    /// <summary>
+    /// A Orginization focused around the education of students.
+    /// </summary>
+    public class School : Organization, IGradable
+    {
+        /// <summary>
+        /// The name of the district this School is part of.
+        /// </summary>
+        public string DistrictName { get; private set; }
+
+        /// <summary>
+        /// The level of education this school falls under
```

Comment:
It's a nit, but some of the XML comments do not end in periods. For API doc consistency, it would be good for each to be a complete sentence and conclude with a period.